doc/asciidoc/consumer/configuration.adoc

```diff
@@ -20,4 +20,85 @@ kafka.value.deserializer=org.apache.kafka.common.serialization.ByteArrayDeserial
 {environment}.enabled=<true/false, default=true>
 ----
 
-See the https://kafka.apache.org/documentation/#brokerconfigs[Apache Kafka documentation] for details on these settings.
\ No newline at end of file
+See the https://kafka.apache.org/documentation/#brokerconfigs[Apache Kafka documentation] for details on these settings.
+
+==== Multi Database Support
+
+Neo4j 4.0 Enterprise has https://neo4j.com/docs/operations-manual/4.0/manage-databases/[multi-tenancy support],
+in order to support this feature you can set for each database instance a configuration suffix with the following pattern
+`to.<DB_NAME>` to the properties in your neo4j.conf file.
+
+Following the list of new properties that allows to support multi-tenancy:
+
+----
+streams.sink.topic.cypher.<TOPIC_NAME>.to.<DB_NAME>=<CYPHER_QUERY>
+streams.sink.topic.cdc.sourceId.to.<DB_NAME>=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.cdc.schema.to.<DB_NAME>=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.pattern.node.<TOPIC_NAME>.to.<DB_NAME>=<NODE_EXTRACTION_PATTERN>
+streams.sink.topic.pattern.relationship.<TOPIC_NAME>.to.<DB_NAME>=<RELATIONSHIP_EXTRACTION_PATTERN>
+streams.sink.enabled.to.<DB_NAME>=<true/false, default=true>
+streams.procedures.enabled.<DB_NAME>=<true/false, default=true>
+----
+
+This means that for each db instance you can specify if:
+* use the source connector
+* the routing patterns
+* use the procedures
+
+So if you have a instance name `foo` you can specify a configuration in this way:
+
+----
+streams.sink.topic.cypher.<TOPIC_NAME>.to.foo=<CYPHER_QUERY>
+streams.sink.topic.cdc.sourceId.to.foo=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.cdc.schema.to.foo=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.pattern.node.<TOPIC_NAME>.to.foo=<NODE_EXTRACTION_PATTERN>
+streams.sink.topic.pattern.relationship.<TOPIC_NAME>.to.foo=<RELATIONSHIP_EXTRACTION_PATTERN>
+streams.sink.enabled.to.foo=<true/false, default=true>
+streams.procedures.enabled.foo=<true/false, default=true>
+----
+
+The old properties:
+
+----
+streams.sink.topic.cypher.<TOPIC_NAME>=<CYPHER_QUERY>
+streams.sink.topic.cdc.sourceId=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.cdc.schema=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+streams.sink.topic.pattern.node.<TOPIC_NAME>=<NODE_EXTRACTION_PATTERN>
+streams.sink.topic.pattern.relationship.<TOPIC_NAME>=<RELATIONSHIP_EXTRACTION_PATTERN>
+streams.sink.enabled=<true/false, default=true>
+----
+
+are still valid and they refer to Neo4j's default db instance.
```

Comment:
...which is usually called "neo4j", but can be controlled by separate Neo4j system configuration.

Reply:
Done